FeedAPIChallenge/Feed API/FeedImagesMapper.swift

```diff
@@ -0,0 +1,48 @@
+//
+//  FeedImagesMapper.swift
+//  FeedAPIChallenge
+//
+//  Created by Chen Yi-Wei on 2021/9/3.
+//  Copyright © 2021 Essential Developer Ltd. All rights reserved.
+//
+
+import Foundation
+
+internal struct FeedImagesMapper {
+	private struct Root: Decodable {
+		let items: [Item]
+
+		var feedImages: [FeedImage] {
+			items.map { $0.feedImage }
```

Comment:
Suggestion: another way of using the `map` function is to use key paths:
```suggestion
			items.map(\.feedImage)
```